appsec/tests/extension/inc/request_replayer.inc

```diff
@@ -0,0 +1,151 @@
+<?php
```

Comment:
The mock_helper already has an http server running on 1826 that can echo back traces: https://github.com/DataDog/dd-trace-php/blob/master/appsec/tests/mock_helper/mock_dd_agent.cc#L55